schemas/triniti/ovp.jwplayer/event/media-synced/1-0-1.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<pbj-schema xmlns="http://gdbots.io/pbj/xsd"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://gdbots.io/pbj/xsd http://gdbots.io/pbj/xsd/schema.xsd">
+
+  <schema id="pbj:triniti:ovp.jwplayer:event:media-synced:1-0-1">
+    <fields>
+      <field name="node_ref" type="identifier" required="true">
+        <php-options>
+          <imports>use Gdbots\Schemas\Ncr\NodeRef;</imports>
+          <class-name>NodeRef::class</class-name>
+        </php-options>
+        <js-options>
+          <imports>import NodeRef from '@gdbots/schemas/gdbots/ncr/NodeRef';</imports>
+          <class-proto>NodeRef</class-proto>
+        </js-options>
+      </field>
+      <field name="jwplayer_media_id" type="string" pattern="^[\w-]+$"/>
+      <field name="jwplayer_caption_keys" type="string" pattern="^[\w]+$" rule="map">
+        <description>
+          The keys of the jwplayer captions that were generated.
+        </description>
+      </field>
+      <field name="thumbnail_image_id" type="identifier">
```

Comment:
@efjacobson if we can just store the asset ref I'd prefer that.  `thumbnail_ref` -> noderef